R/conclass.R

```diff
@@ -640,14 +648,48 @@ Conos <- setRefClass(
     },
 
 
-    plotGraph=function(color.by='cluster', clustering=NULL, groups=NULL, colors=NULL, gene=NULL, plot.theme=NULL, ...) {
+    plotGraph=function(color.by='cluster', clustering=NULL, groups=NULL, colors=NULL, gene=NULL, plot.theme=NULL, subcluster=NULL, plot.na=NULL, keep.limits=NULL, ...) {
       if(class(embedding)[1] == "uninitializedField") {
         embedGraph();
       }
 
+      if(!is.null(subcluster)) {
```

Comment:
I agree that subgroup (subcluster) doesn't do much, and maybe we should change it again.
However, using subgroup it sets plot.na=F and keep.limits=T. Keep.limits maintains original axis limits, preferable when you want to show subgroup in the same space as initial graph.

Most importantly, you can't do "groups" and "gene" at the same time right now. But with subgroup you can:
con$plotGraph(subgroup=T-cell, gene='gene')

Also, plotting subgroups can be done like this
con$plotGraph(subgroup=T-cell) #named factor
instead of
con$plotGraph(groups=annotation, subcluster='T-cell') 
or
con$plotGraph(groups=T-cell, plot.na=F, keep.limits=T)

Would it be an idea to include both "subgroup" and "cluster"? i.e.
con$plotGraph(subgroup=annotation, cluster="CD4 T-cell", gene="gene")

For you and I, it isn't difficult to include plot.na and keep.limits in our code each time, but for unskilled people it may just be more noice.